showcase/config/index-head-config.js

```diff
@@ -32,7 +32,8 @@ module.exports = {
 		{ rel: "icon", type: "image/png", sizes: "192x192", href: "assets/images/app-icons/android-icon-192x192.png" },
 		{ rel: "icon", type: "image/png", sizes: "32x32", href: "assets/images/app-icons/favicon-32x32.png" },
 		{ rel: "icon", type: "image/png", sizes: "96x96", href: "assets/images/app-icons/favicon-96x96.png" },
-		{ rel: "icon", type: "image/png", sizes: "16x16", href: "assets/images/app-icons/favicon-16x16.png" }
+		{ rel: "icon", type: "image/png", sizes: "16x16", href: "assets/images/app-icons/favicon-16x16.png" },
+		{ rel: "stylesheet", type: "text/css", href: "assets/stark-core/css/pre-load-style.css" }
```

Comment:
Can you add the same comment that was added for this CSS in the `index.html`? So it is clear why we this is added here ;)